test/unit/models/hospitalAdministrator.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const assert = require('assert');
+const helper = require('../../helper');
+const models = require('../../../models');
+
+describe('models.HospitalAdministrator', function () {
+  beforeEach(async function() {
+    await helper.loadFixtures(['hospital']);
+  });
+
+  it('creates a new HospitalAdministrator record', async function() {
+    const hospitalAdmin = await models.HospitalAdministrator.create({
+      HospitalId: '7f666fe4-dbdd-4c7f-ab44-d9157379a680',
```

Comment:
Here we see the uuid of the fixture hospital record referenced in creating the new HospitalAdministrator.